commune/urls.py

```diff
@@ -23,6 +23,7 @@
     path('admin/', admin.site.urls),
     path('', views.main_page, name='Main_Page'),
     path('chores/', views.chores, name="chores"),
+    path('commune/', views.commune, name="commune"),
```

Comment:
I think you need to delete the above line:
    path('chores/', views.chores, name="chores"),

Reply:
fixed.